R/bm-row-group-size.R

```diff
@@ -0,0 +1,68 @@
+#' Benchmark effect of parquet row group size
+#'
+#' @section Parameters:
+#' * `source` A known-file id, or a file path to read in
+#' * `chunk_size` Number of rows to write in each row group. Suggested sizes:
+#'     `chunk_size = list(NULL, 10000L, 100000L, 1000000L)`
+#'
+#' @export
+row_group_size <- Benchmark(
+  "row_group_size",
+  setup = function(source = "fanniemae_2016Q4",  # TODO implement more sources
+                   chunk_size = NULL) {
```

Comment:
What the reason that this is being left as `NULL`?

Reply:
That's the default for `write_parquet()`. `match.arg()` only works for character vectors, so there's not a great way to set a list of options/defaults here afaik; it's something to be aware of when using `default_params()` with this benchmark. We could write a more flexible version of `match.arg()` to make `default_params()` better, but it would confuse anybody new to this code. Or maybe we could store non-character defaults as attributes or something?